src/CMakeLists.txt

```diff
@@ -733,6 +733,13 @@ add_subdirectory(erasure-code)
 
 # Support/Tools
 if(WITH_TESTS)
+  find_program(CMAKE_MEMORYCHECK_COMMAND valgrind)
+  message(STATUS "Executing test suite with Valgrind (${CMAKE_MEMORYCHECK_COMMAND})")
+  set(MEMORYCHECK_COMMAND_OPTIONS "--error-exitcode=1 --leak-check=full --suppressions=${CMAKE_SOURCE_DIR}/qa/valgrind.supp")
```

Comment:
nit: not sure we need to repeat the `--suppression` here as `MEMORYCHECK_SUPPRESSION_FILE` is defined below.